views/teams.tmpl

```diff
@@ -0,0 +1,47 @@
+{{define "content"}}
```

Comment:
2 spaces rather than tabs. I see I haven't included tmpl files in the .editorconfig, I shall correct this now